### ISSUE TITLE
Labels should only be used for form controls, and should be associated with a form element

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1088]
- [x] Labels should only be used for form controls
- [x] Label should be associated with a form element implicitly or explicitly using layout (wrapping the form element in the label) or using the `for/htmlFor`, `aria-label`, or `aria-labelledby` attributes (`for` attr is preferred but likely all strategies will be necessary --depending on the application).
